static/absolute/js/searchPage.js

```diff
@@ -128,12 +128,25 @@ async function refreshUI() {
  * For now, this function deals with hard-coded data, but this
  * can be used as a template for when we get data the Places Library results.
  */
-function showRestaurants(restaurantsResponse) {
-  const allRestaurants = restaurantsResponse.data;
+async function showRestaurants(restaurantResponse) {
   const restaurantContainer = document.getElementById("restaurant-container");
-
   restaurantContainer.innerHTML = "";
 
+  if (restaurantResponse.status !== 200) {
+    let restaurantErrorMessage = document.createElement("p");
+    restaurantErrorMessage.classList.add("search-instructions");
+    let messageText =
+      restaurantResponse.data === "ZERO_RESULTS"
+        ? "We could not find any restaurants. Check to make sure you are using the correct address."
+        : "Something went wrong when searching for restaurants.";
+    restaurantErrorMessage.appendChild(document.createTextNode(messageText));
```

Comment:
Ved made some changes to the places API that may affect this. I suggest you rebase and run through it when he merges avoid any future issues

Reply:
I saw. I already plan to wait for his changes and rebase.